go/enclave/rollupchain/rollup_chain.go

```diff
@@ -270,24 +270,45 @@ func (rc *RollupChain) findRoundWinner(receivedRollups []*obscurocore.Rollup, pa
 	return headRollup
 }
 
-func (rc *RollupChain) process(rollup *obscurocore.Rollup, stateDB *state.StateDB, depositTxs []*common.L2Tx) (gethcommon.Hash, []*types.Receipt, []*types.Receipt) {
-	txReceipts := evm.ExecuteTransactions(rollup.Transactions, stateDB, rollup.Header, rc.storage, rc.obscuroChainID, 0)
-	if len(rollup.Transactions) != len(txReceipts) {
-		panic("Sanity check. All transactions that are included in a rollup must be executed and produce a receipt.")
+func (rc *RollupChain) process(rollup *obscurocore.Rollup, txs obscurocore.L2Txs, stateDB *state.StateDB) (gethcommon.Hash, obscurocore.L2Txs, []*types.Receipt, []*types.Receipt) {
```

Comment:
Not a new issue but this method could do with a small comment just to say high-level what it's used for